src/pages/publication/contributors_tab/AddContributor.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { FC, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { Button } from '@material-ui/core';
+
+import Modal from '../../../components/Modal';
+import { Authority } from '../../../types/authority.types';
+import AddContributorModal from './components/AddContributorModal';
+
+const AddContributor: FC = () => {
+  const { t } = useTranslation('publication');
+
+  const [addedAuthors, setAddedAuthors] = useState<Authority[]>([]);
+  const [open, setOpen] = useState(false);
+
+  const handleOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const addAuthor = (author: Authority) => {
+    setAddedAuthors([...addedAuthors, author]);
+    handleClose();
+  };
+
+  return (
+    <>
+      {addedAuthors && addedAuthors.map(author => <h4 key={author.systemControlNumber}>{author.name}</h4>)}
```

Comment:
Dette er en del av neste feature, så se bort ifra denne linjen (ble med kun for å teste at vi får lagt til forfattere etter å ha valgt de i modalen)